rasa/nlu/classifiers/diet_classifier.py

```diff
@@ -192,6 +192,15 @@ def _check_config_parameters(self) -> None:
             self.component_config
         )
 
+        if (
+            self.component_config[MASKED_LM]
+            and not self.component_config[NUM_TRANSFORMER_LAYERS]
```

Comment:
```suggestion
            and self.component_config[NUM_TRANSFORMER_LAYERS] == 0
```
Let's use this notation as it is an integer.